tldr.py

```diff
@@ -120,12 +119,18 @@ def have_recent_cache(command, platform):
         return False
 
 
-def get_page_for_platform(command, platform):
+def get_page_url(platform, command, remote=None):
```

Comment:
Why not make `remote=DEFAULT_REMOTE` itself as the default param ? Then you don't need to make the check.

Reply:
Because using mutable types as default value is not good. https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html